src/pages/TipsCalculator/TipsCalculator.jsx

```diff
@@ -1,10 +1,60 @@
 import React from 'react';
 import Header from '../../components/Header';
+import './TipsCalculator.css';
+import BillContainer from '././Components/BillContainer';
+import logo from '../../assets/tips-calc-img/logo.svg';
+import TipsContainer from '././Components/TipsContainer';
+import PeopleContainer from '././Components/PeopleContainer';
+import RecapContainer from '././Components/RecapContainer';
+import { useEffect, useState } from 'react';
 
 function TipsCalculator() {
+  const [bill, setBill] = useState(null);
+  const [tip, setTip] = useState(null);
+  const [nbPeople, setNbPeople] = useState(null);
+  const [tipAmount, setTipAmount] = useState(null);
+  const [totalAmount, setTotalAmount] = useState(null);
+
+  useEffect(() => {
+    if (bill > 0 && tip > 0 && nbPeople > 0) {
+      setTipAmount(parseFloat((bill * tip) / 100 / nbPeople).toFixed(2));
+      setTotalAmount(
+        parseFloat((tipAmount * nbPeople + bill / nbPeople).toFixed(2))
+      );
+    }
+    if (bill === 0 || tip === 0 || nbPeople === 0) {
```

Comment:
If you put 0 as default value in useState you can maybe do
`if (!bill || !tip || !nbPeople)` here.